sdk/monitor/azquery/CHANGELOG.md

```diff
@@ -1,14 +1,9 @@
 # Release History
 
-## 0.4.1 (Unreleased)
-
-### Features Added
+## 1.0.0 (2-7-23)
```

Comment:
```suggestion
## 1.0.0 (2023-02-07)
```

Reply:
fixed in [e48b6c9](https://github.com/Azure/azure-sdk-for-go/pull/19946/commits/e48b6c941d9c1e27bc3f3c0aea767646b1694985)